packages/twenty-front/src/modules/workspace-invitation/hooks/useCreateWorkspaceInvitation.ts

```diff
@@ -0,0 +1,26 @@
+import { useSendInvitationsMutation } from '~/generated/graphql';
+import { useRecoilState } from 'recoil';
+import { workspaceInvitationsState } from '../states/workspaceInvitationsStates';
+import { SendInvitationsMutationVariables } from '../../../generated/graphql';
+
+export const useCreateWorkspaceInvitation = () => {
+  const [sendInvitationsMutation] = useSendInvitationsMutation();
+
+  const [, setWorkspaceInvitations] = useRecoilState(workspaceInvitationsState);
```

Comment:
useSetRecoilState!

Unfortunately, if you use useRecoilState you are doing an underlying useRecoilValue and useSetRecoilState even if you don't use the value